internal/db/schema/migrations/20230322114550_add_links_table.sql

```diff
@@ -0,0 +1,24 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE links
+(
+    id                              UUID PRIMARY KEY DEFAULT gen_random_uuid(),
```

Comment:
```suggestion
    id                              UUID PRIMARY NOT NULL,
```

It is better to generate ids in the domain, instead of relying on database generation. This is what we've been doing in the last tables we've created.

Reply:
> I added some request changes. We can discuss it between the team. We need to agree in how to generate ids. I prefer to do it inside the domain and not let the DB to generate for us.

I dont agree with that. But I will change it. I don't see a good reason.

Reply:
Generating IDs in the domain isolates you from repository implementation, You can mock the repo or reimplement it in another persistence layer like Redis, Mongo, etc.. and the  id generation will not change if done in the domain.